src/datafactory/azext_datafactory/generated/custom.py

```diff
@@ -0,0 +1,884 @@
+                                         factory_name):
+    return client.list_by_factory(resource_group_name=resource_group_name,
+                                  factory_name=factory_name)
+
+
+def datafactory_integration_runtime_show(client,
+                                         resource_group_name,
+                                         factory_name,
+                                         integration_runtime_name,
+                                         if_none_match=None):
+    return client.get(resource_group_name=resource_group_name,
+                      factory_name=factory_name,
+                      integration_runtime_name=integration_runtime_name,
+                      if_none_match=if_none_match)
+
+
+def datafactory_integration_runtime_linked_integration_runtime_create(client,
+                                                                      resource_group_name,
+                                                                      factory_name,
+                                                                      integration_runtime_name,
+                                                                      name=None,
+                                                                      subscription_id=None,
+                                                                      data_factory_name=None,
+                                                                      data_factory_location=None):
+    return client.create_linked_integration_runtime(resource_group_name=resource_group_name,
+                                                    factory_name=factory_name,
+                                                    integration_runtime_name=integration_runtime_name,
+                                                    name=name,
+                                                    subscription_id=subscription_id,
+                                                    data_factory_name=data_factory_name,
+                                                    data_factory_location=data_factory_location)
+
+
+def datafactory_integration_runtime_managed_create(client,
+                                                   resource_group_name,
+                                                   factory_name,
+                                                   integration_runtime_name,
+                                                   if_match=None,
+                                                   description=None,
+                                                   type_properties_compute_properties=None,
+                                                   type_properties_ssis_properties=None):
+    if isinstance(type_properties_compute_properties, str):
+        type_properties_compute_properties = json.loads(type_properties_compute_properties)
+    if isinstance(type_properties_ssis_properties, str):
+        type_properties_ssis_properties = json.loads(type_properties_ssis_properties)

```

Comment:
Add `type=validate_file_or_dict` in your `CLIArgumentType` definition in `_params.py`, then here you can use the parameters directly as they are already json loaded.

Reply:
This should also been resolved now, please review again. Thx